game-core/src/main/java/games/strategy/triplea/ai/pro/util/MoveBatcher.java

```diff
@@ -1,35 +1,35 @@
 package games.strategy.triplea.ai.pro.util;
 
+import games.strategy.engine.data.MoveDescription;
 import games.strategy.engine.data.Route;
 import games.strategy.engine.data.Unit;
 import java.util.ArrayList;
-import java.util.Collection;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-import javax.annotation.Nullable;
 
 /**
  * MoveBatcher implements an algorithm that batches AI moves together, reducing the number of
  * distinct moves the AI does (and thus the visual delay) by combining moves together.
  */
 public class MoveBatcher {
+  // TODO: Merge/replace with MoveDescription.
```

Comment:
nit: should include a 'grep' or a 'grouping' token with the TODO. Something so we can track it back to the original task, we've a problem of decades old TODO comments that do not make a lot of sense and decided to avoid that would be by marking/grouping TODO comments so we can find them. An issue, or a PR number would probably be good choices.

Reply:
Done.